packages/newspringchurchapp/src/prayer/PrayerActionMenuCard/PrayerActionMenuCard.js

```diff
@@ -0,0 +1,78 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Card,
+  CardContent,
+  H4,
+  styled,
+  Touchable,
+  Icon,
+  PaddedView,
+} from '@apollosproject/ui-kit';
+
+import NSIcon from 'newspringchurchapp/src/ui/NSIcon';
+
+const StyledCard = styled(({ expandedHeight }) => ({
+  marginHorizontal: 0,
+  marginVertical: 0,
+  height: expandedHeight,
+}))(Card);
+
+const StyledCardContent = styled(() => ({
+  alignItems: 'center',
+}))(CardContent);
+
+const StyledPaddedView = styled(() => ({
+  alignItems: 'center',
+}))(PaddedView);
+
+const StyledTouchable = styled(({ theme }) => ({
+  alignItems: 'center',
+  padding: theme.sizing.baseUnit * 1.5,
+}))(Touchable);
+
+const PrayerActionMenuCard = memo(
+  ({ exitPrayer, savePrayer, advancePrayer, expandedHeight }) => (
+    <>
```

Comment:
Why is this needed?

Reply:
Took them out. They were there from a previous version of this file.